_sass/minima/_base.scss

```diff
@@ -101,9 +101,6 @@ h1,
   font-size: 30px;
   margin-top: 1em;
   text-transform: uppercase;
-  @include media-query($on-palm) {
-    margin-top: 2em;
-  }
```

Comment:
I'm not sure why this was here.

Reply:
I think at some point the lang bar was overlapping the header in low resolution devices